http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/TestUtils.scala

```diff
@@ -21,6 +22,29 @@ object TestUtils extends Matchers {
     }
   }
 
+  implicit val leonardoExceptionEq = {
+    new Equality[LeoException] {
+      def areEqual(a: LeoException, b: Any): Boolean =
+        b match {
+          case bb: LeoException =>
+            println("comparinggggg")
+            a.message == bb.message && a.statusCode == bb.statusCode && a.cause == bb.cause
```

Comment:
This looks right but I just wonder if there's a built-in way to check `Exception` equality that we could leverage instead of rolling our own in case there are edge cases we may not be aware of.

Reply:
I don't think there's a "built-in" way to check `Exception`...